.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Here are some checklists you may like to use. Use your judgement.
+
+If you're new to the codebase, or you have questions, CONTRIBUTING has many answers.
+-->
+Pre-submit checklist:
+- [ ] Key commits have useful messages
```

Comment:
Apart from sometimes just being lazy nobody writes useless commit messages _intentionally_ so I think it would be good to be more specific what the expectation towards a "useful" commit message is. We might be able to just re-use such a guideline/definition from some other project or summarize something ourselves. But would be good to have concrete expectations/requirements that can be linked to.

This shouldn't be anything very involved anyhow. What i think of are things like:
- Write a single line with a summary followed by an empty line and a paragraph with details (where applicable)
- Describe _why_, not only _what_ (Write "using fork of X *because* \<reasons\>" instead of "using for of X")
- Reference external sources where applicable (Write "using fork of X until \<PR URL> is merged")

Reply:
There's some more detail in `CONTRIBUTING`, maybe I should make the connection more explicit. I wanted to avoid shoving lots of detail in the checklist!